core/src/main/scala/org/apache/pekko/persistence/jdbc/state/DurableStateQueries.scala

```diff
@@ -92,6 +96,16 @@ import slick.jdbc.{ H2Profile, JdbcProfile, OracleProfile, PostgresProfile, SQLS
     durableStateTable.filter(_.persistenceId === persistenceId).delete
   }
 
+  /**
+   * Deletes a particular revision of an object based on its persistenceId.
+   * This revision may no longer exist and if so, no delete will occur.
+   *
+   * @since 1.1.0
+   */
+  private[jdbc] def deleteBasedOnPersistenceIdAndRevision(persistenceId: String, revision: Long) = {
+    durableStateTable.filter(r => r.persistenceId === persistenceId && r.revision === revision).delete
```

Comment:
we could use `selectFromDbByPersistenceId(persistenceId).filter(_. revision  === revision).delete`. filter mutiple columns in one line wasn't recommended? at least i didn't see any usage in the official doc: https://scala-slick.org/doc/stable/queries.html#sorting-and-filtering

the parameter should be Rep[String] too because you are defining a SQL.

Reply:
I've used filters with multiple conditions many times with no issues. I can change this nonetheless.